src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String...args) {
+        printFigures(generateRandomFigureList());
+    }
+
+    private static List<Figure> generateRandomFigureList() {
+        int initialCapacity = (int) ((Math.random() + 3) * 5);
+        List<Figure> result = new ArrayList<>(initialCapacity);
+        int iterator = 0;
+        while (iterator < initialCapacity) {
+            switch ((int) (Math.random() * 5)) {
+                case 0 :
+                    result.add(new Circle());
+                    break;
+                case 1 :
+                    result.add(new IsoscelesTrapezoid());
```

Comment:
it is hard to understand, that an empty constructor will be generated figures with different parameters. 
Let's generate all values in this method and pass to the constructor as arguments.

Reply:
Won't it reduce the readability? Some classes require a low of paremeters

Reply:
My solution involves implementing a separate method for each `Figure` generation in `FigureGenerator`